src/schema/serde.src.rs

```diff
@@ -23,18 +23,18 @@ pub fn from_file(path: PathBuf) -> Result<Definition, Error> {
     "json" => {
       let reader = BufReader::new(file);
       let data: SerdeDefinition = try!(serde_json::from_reader(reader));
-      Ok(try!(data.to_definition()))
+      Ok(try!(data.into_definition()))
```

Comment:
Ok, here was the debate.

Clippy said that to_* is normally associated with a &self.

I chose `into` because I thought of that first. However I think `as` would have worked better.
We could impl `Into`.